samples/x-iac-pipeline/app/cloudbuild.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+steps:
+  - id: 'branch name'
+    name: 'alpine'
+    entrypoint: 'sh'
+    args:
+      - '-c'
+      - |
+          echo "***********************"
+          echo "$BRANCH_NAME"
+          echo "***********************"
+  - name: gcr.io/cloud-builders/gcloud-slim
+    id: fetch-token
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+          # Fetch a GCP bearer token
+          gcloud auth print-access-token > /workspace/token
+  - name: 'gcr.io/cloud-builders/mvn'
+    id: deploy-api
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+          # Deploy the API
+          cd httpbin
+          mvn install -Phttpbin -Dorg=${_APIGEE_ORG} -Denv=${_APIGEE_ENV} -Dbearer=$(cat /workspace/token)
```

Comment:
can we use `$(gcloud auth print-access-token)` instead of writing and reading the token from the fs?

Reply:
The mvn builder image does not contain gcloud.
If we would want to run this in one step we would have to create a custom image ourselves.
Not sure whether that would be worth the trade-off.
What do you reckon?

Reply:
ah fair enough. never mind then